pydm/widgets/image.py

```diff
@@ -91,7 +91,8 @@ def receiveImageWaveform(self, new_waveform):
       self.image_waveform = new_waveform.reshape((int(self.image_width),-1), order='F')
     elif len(new_waveform.shape) == 2:
       self.image_waveform = new_waveform
-    self.data_max_int = np.iinfo(self.image_waveform.dtype).max
+    self.data_max_int = np.amax(self.image_waveform) # take the max value of the recieved image
+    self.setColorMap() # to update the colormap immediately
```

Comment:
The changes to the image widget are great!  Your method is much better than what I did before.